app/views/api/asciicasts/show.html.slim

```diff
@@ -19,6 +19,14 @@ javascript:
       if (e.data == 'asciicast:play' || e.data[0] == 'asciicast:play') {
         player.play();
       }
+
+      // reveal.js support
+
+      if (e.data == 'slide:start') {
```

Comment:
My point is, the embed `.js` and the `<script>` in the iframe page don't have a simple way to detect reveals, so it probably needs to be passed down explicitly from your slides markup.